### PR TITLE
feat(fmt): quick-fix <> `@@schema` attribute

### PR DESCRIPTION
**Pull request by Druue**
closes https://github.com/prisma/language-tools/issues/1329

### CODE REVIEW

File: prisma-fmt/src/code_actions/multi_schema.rs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    let span_diagnostics =
+        match super::diagnostics_for_span(schema, &params.context.diagnostics, enumerator.ast_enum().span()) {
+            Some(sd) => sd,
+            None => return,
+        };
+
+    let diagnostics =
+        match super::filter_diagnostics(span_diagnostics, "This model is missing an `@@schema` attribute.") {
```
Comment (pimeys) [inline prisma-fmt/src/code_actions/multi_schema.rs:87]:
```suggestion
        match super::filter_diagnostics(span_diagnostics, "This enum is missing an `@@schema` attribute.") {
```

?

File: prisma-fmt/tests/code_actions/scenarios/multi_schema_one_model_one_enum/result.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+[
```
Comment (pimeys) [inline prisma-fmt/tests/code_actions/scenarios/multi_schema_one_model_one_enum/result.json:1]: Do we miss the enum fix here?

File: psl/parser-database/src/walkers/enum.rs [hunk 3]
```diff
@@ -46,6 +48,16 @@ impl<'db> EnumWalker<'db> {
         (0..self.ast_enum().values.len()).map(move |idx| self.walk((self.id, idx)))
     }
 
+    /// How fields are indented in the enum.
+    pub fn indentation(self) -> IndentationType {
+        IndentationType::default()
```
Comment (pimeys) [inline psl/parser-database/src/walkers/enum.rs:53]: Are we sure about this?

File: psl/parser-database/src/walkers/enum.rs [hunk 4]
```diff
@@ -46,6 +48,16 @@ impl<'db> EnumWalker<'db> {
...
+    }
+
+    /// What kind of newlines the enum uses.
+    pub fn newline(self) -> NewlineType {
+        NewlineType::Unix
```
Comment (pimeys) [inline psl/parser-database/src/walkers/enum.rs:58]: Are we sure about this?

**Review by pimeys — COMMENTED** [review]
a few questions, answer tomorrow or after holidays :)

File: psl/parser-database/src/walkers/enum.rs (hunk 3 above)
Comment (Druue) [reply psl/parser-database/src/walkers/enum.rs:53]: Yeah per convo with @tomhoule yesterday. If it doesn't line up, it should just be a case of re-running the formatter. The future goal is to remove these functions from both enum and model etc and instead ask the editor for this information

File: psl/parser-database/src/walkers/enum.rs (hunk 4 above)
Comment (Druue) [reply psl/parser-database/src/walkers/enum.rs:58]: See above comment for indentation

File: psl/parser-database/src/walkers/enum.rs (hunk 3 above)
Comment (pimeys) [reply psl/parser-database/src/walkers/enum.rs:53]: That is fine for me!

File: psl/parser-database/src/walkers/enum.rs (hunk 4 above)
Comment (pimeys) [reply psl/parser-database/src/walkers/enum.rs:58]: Cool.

File: prisma-fmt/src/code_actions/multi_schema.rs (hunk 1 above)
Comment (Druue) [reply prisma-fmt/src/code_actions/multi_schema.rs]: Sorted

File: prisma-fmt/tests/code_actions/scenarios/multi_schema_one_model_one_enum/result.json (hunk 2 above)
Comment (Druue) [reply prisma-fmt/tests/code_actions/scenarios/multi_schema_one_model_one_enum/result.json:1]: Yep! this was due to enum qf not filtering the correct diagnostic per [above](https://github.com/prisma/prisma-engines/pull/3525#discussion_r1054631400)

**Review by pimeys — APPROVED** [review]
✨

**Pull request merged**